src/containers/course-section/resources-list/ResourcesList.styles.ts

```diff
@@ -9,6 +9,7 @@ export const styles = {
     position: 'relative',
     backgroundColor: 'basic.white',
     borderRadius: '6px',
+    zIndex: isDragging ? 10 : 1,
```

Comment:
Default value for `z-index` is 0, if we can drag only 1 item per time, maybe it's better to set up it like this, should still work, and we don't overload page with redundant high z-index
What do you think?
```suggestion
    zIndex: isDragging ? 1 : 0,
```